docs/src/main/asciidoc/hibernate-orm-guide.adoc

```diff
@@ -113,6 +113,9 @@ To load some SQL statements when Hibernate ORM starts, add a `import.sql` in the
 It contains SQL DML statements (one by line).
 This is useful to have a data set ready for your tests or demos.
 
+WARNING: The `@Transactional` annotation is required on any method that modifies the database. If you forget it you
```

Comment:
This should be something like 'An active transaction is required for every method that modifies the database. You can create a container managed transaction using the @Transactional annotation on any CdI bean.'

As it is written it sounds like you need to put this annotation on every method, when really it will mostly be on the application entry points, and will result in cargo-cult level @Transactional everywhere.

We also should look at implementing a Seam 2 style TX by default in reasteasy.

Reply:
OK I'll modify it.

Reply:
Is this better now?